src/main/java/uk/ac/ebi/ampt2d/accession/sha1/SHA1AccessionGenerator.java

```diff
@@ -15,8 +15,10 @@
  * limitations under the License.
  *
  */
-package uk.ac.ebi.ampt2d.accession;
+package uk.ac.ebi.ampt2d.accession.sha1;
 
+import uk.ac.ebi.ampt2d.accession.AccessioningObject;
+import uk.ac.ebi.ampt2d.accession.SingleAccessionGenerator;
 import uk.ac.ebi.ampt2d.accession.utils.Sha1Util;
```

Comment:
Now that we have a package specific for SHA1 stuff we could move this utility class inside it.